src/components/footer/footer.spec.js

```diff
@@ -23,6 +23,12 @@ describe('Footer Component', () => {
       expect(footer.shadowRoot.querySelectorAll('footer').length).equal(1);
     });
 
+    it('should have two div containers', () => {
+      const elements = footer.shadowRoot.querySelectorAll('div.row > div');
```

Comment:
maybe we could test for something a little more concrete?  Like N number of `<a>` tags to capture each link added?